docker/Jetson/Jetpack4.6/Dockerfile

```diff
@@ -0,0 +1,119 @@
+FROM nvcr.io/nvidia/l4t-pytorch:r32.7.1-pth1.10-py3
+
+ARG MMDEPLOY_VERSION=main
+ENV NVIDIA_VISIBLE_DEVICE all
+ENV NVIDIA_DRIVER_CAPABILITIES all
+ENV CUDA_HOME="/usr/local/cuda"
+ENV PATH="/usr/local/cuda/bin:${PATH}"
+ENV LD_LIBRARY_PATH="/usr/local/cuda/lib64:/usr/local/lib/python3.8/dist-packages/opencv-python.libs/${LD_LIBRARY_PATH}"
+ENV TENSORRT_DIR="/usr/include/aarch64-linux-gnu"
+
+ENV DEBIAN_FRONTEND=nointeractive
+ENV FORCE_CUDA="1"
+
+USER root
+WORKDIR /root/workspace
+
+# install dependencies && reinstall python3.8
+RUN  apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 42D5A192B819C5DA &&\
+     apt-get remove python3 &&\ 
+     apt-get update &&\ 
+     apt-get install -y vim wget libspdlog-dev libssl-dev libpng-dev pkg-config libhdf5-100 libhdf5-dev patch --no-install-recommends\
+                        python3.8 python3.8-dev python3.8-pip --no-install-recommends &&\
+     python3.8 -m pip install --upgrade --no-cache-dir setuptools packaging 'Cython<3' wheel &&\
+     python3.8 -m pip install --no-cache-dir --verbose wget psutil numpy &&\
```

Comment:
why need to install these packages?

Reply:
>      python3.8 -m pip install --upgrade --no-cache-dir setuptools packaging 'Cython<3' wheel &&\
     python3.8 -m pip install --no-cache-dir --verbose wget psutil numpy &&\

this package for build the pytorch, I referenc from l4t repo